### ISSUE TITLE
elasticsearch returns a mapper_parsing_exception error

### ISSUE THREAD

**Issue by maxlath** [ryanramage/couch2elastic4sync#3]
hi!
Trying to copy a couchdb database to elasticsearch using either `couch2elastic4sync load` or `couch2elastic4sync`, I get this error:
```javascript
 { type: 'mapper_parsing_exception',
   reason: 'Field [_id] is a metadata field and cannot be added inside a document. Use the index API request parameters.' }
```
`_id` being a standard couchdb document attribute, how do you manage to overpass this issue? I guess it could be simple to edit the attribute key in the pipeline but I wonder how you didn't encounter this error, did I miss something?

**Comment by ryanramage** [comment]
would you be able to post the document as stored in couchdb?

**Comment by gesellix** [comment]
This is a changed behaviour when using Elasticsearch 2.x. See https://github.com/elastic/elasticsearch/pull/14003

**Comment by ryanramage** [comment]
9232076 commit was actually for issue #3

**Comment by ryanramage** [comment]
Have a look at https://github.com/ryanramage/couch2elastic4sync/blob/92320764690136b527b5897ed9d7d7cf6962523b/lib/remove-meta.js and let me know if that catches all the meta fields that cause problems. If so, I will merge and publish

**Comment by gesellix** [comment]
would it make sense to map the couchdb `_id` to elasticsearch `id` before removing it?

**Comment by gesellix** [comment]
ah. forget my last question, it's already implemented that way :-)

**Comment by gesellix** [comment]
@ryanramage your change works for me using Elasticsearch 2.1.1.

**Comment by ryanramage** [comment]
great. I have published as 2.0.0

**Comment by maxlath** [comment]
it's working, thanks! :)
